src/include/duckdb/parser/expression/lambda_expression.hpp

```diff
@@ -19,9 +19,9 @@ namespace duckdb {
 //! Lambda expressions are written in the form of "capture -> expr", e.g. "x -> x + 1"
 class LambdaExpression : public ParsedExpression {
 public:
-	LambdaExpression(unique_ptr<ParsedExpression> lhs, unique_ptr<ParsedExpression> rhs);
+	LambdaExpression(vector<unique_ptr<ParsedExpression>> lhs, unique_ptr<ParsedExpression> rhs);
```

Comment:
LambdaExpression is also used for the JSON `->` arrow operator, in which case `lhs` and `rhs` make more sense.
But since this is called `LambdaExpression`, I agree, we should rename to something more descriptive.

Reply:
`args` sounds good, but the `rhs` is more an expression than a result, no? Maybe call it `expr`?

Reply:
I will rename to `params` and `expr`, to stay consistent with the [python doc](https://docs.python.org/3/reference/expressions.html#lambda), as suggested.